frontend/src/app/api/mock/APIMockResponse.json

```diff
@@ -2,7 +2,7 @@
   "data": [
     {
       "agency": "US-ABC",
-      "category": "continuation",
+      "category": "<======== THIS IS MOCK DATA ================>",
```

Comment:
Nit: This isn't a valid value (category is an enum), so this might lead to some weird behavior / need to account for an impossible scenario. You could set the category explanation on the next line to this.

Reply:
I wanted it to actually show on the page and I don't believe `category_explanation` is one of the fields we will be seeing. I think the mock data is just for display for folks who want to see the results show quickly without having to start the api for a real call.

![image](https://github.com/HHS/simpler-grants-gov/assets/93001277/55df4a63-f606-406b-bf99-90365e85afdc)